src/components/layouts/Footer.tsx

```diff
@@ -20,7 +20,7 @@ const hoverStyles = {
 
 export const Footer: FC = () => {
   return (
-    <Flex mt={{ base: 4, lg: 8 }} direction={{ base: 'column', lg: 'row' }}>
+    <Flex mt={{ base: 4, lg: 8 }} direction={{ base: 'column', lg: 'row' }} position='sticky' top='100%'>
```

Comment:
we should add a `margin-bottom` here as we're losing it with the new position

Reply:
@nhsz Curious what browser you're using?
I'm not seeing this... I'm getting what is shown in the snapshot above and below where there is space below the footer from `my={{ base: 4, md: 7 }}` applied to the Layout container

<img width="603" alt="image" src="https://user-images.githubusercontent.com/54227730/208189583-cfaa9bed-f35e-468e-9704-881d3ce42122.png">

Reply:
Chrome

![deploy-preview-188--melodious-puffpuff-8e1109 netlify app_](https://user-images.githubusercontent.com/948922/208191280-eed8aaf0-d42c-4205-882e-4a443275a661.png)

Reply:
Okay thanks, investigating